.github/workflows/tests.yml

```diff
@@ -0,0 +1,71 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.14', '1.15', '1.16']
```

Comment:
mind leaving 1.13 as long as u-root is on 1.13?

Reply:
sure

Reply:
done